dashboard/app/models/level.rb

```diff
@@ -349,6 +351,27 @@ def strip_name
     self.name = name.to_s.strip unless name.nil?
   end
 
+  def log_changes
+    log = JSON.parse(audit_log || "[]")
+
+    # serialize 'properties' changes
+    changed = self.changed
+    if changed.include?('properties')
+      changed.delete('properties')
```

Comment:
I'm not sure I understand why we're deleting the 'properties' key here.  Note that `changed = self.changed` is a reference assignment so we're also mutating `self.changed`.